CI failing due to unused dependency in package.datascience-ui.dependencies.json

See https://dev.azure.com/ms/vscode-python/_build/results?buildId=58817&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&s=ff05ad62-bb9a-53b6-ce9f-72f329a63e7c&t=095e52e8-3628-5550-ec3a-33db25a99da4&l=95:

```
> python@2020.2.0-dev checkDependencies /home/vsts/work/1/s
> gulp checkDependencies

[22:04:40] Using gulpfile ~/work/1/s/gulpfile.js
[22:04:40] Starting 'checkDependencies'...
[22:04:40] Starting 'checkNativeDependencies'...
[22:04:47] Finished 'checkNativeDependencies' after 6.77 s
[22:04:47] Starting 'check-datascience-dependencies'...
Remove the unused 'memoize-one' dependencies from package.datascience-ui.dependencies.json
[22:07:04] 'check-datascience-dependencies' errored after 2.28 min
[22:07:04] Error: Remove the unused 'memoize-one' dependencies from package.datascience-ui.dependencies.json
    at throwAndLogError (/home/vsts/work/1/s/gulpfile.js:468:15)
    at checkDatascienceDependencies (/home/vsts/work/1/s/gulpfile.js:462:9)
[22:07:04] 'checkDependencies' errored after 2.4 min
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! python@2020.2.0-dev checkDependencies: `gulp checkDependencies`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the python@2020.2.0-dev checkDependencies script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/vsts/work/1/.npm/_logs/2020-01-16T22_07_04_590Z-debug.log

##[debug]Exit code 1 received from tool '/bin/bash'
##[debug]STDIO streams have closed for tool '/bin/bash'
##[error]Bash exited with code '1'.
##[debug]Processed: ##vso[task.issue type=error;]Bash exited with code '1'.
##[debug]task result: Failed
##[debug]Processed: ##vso[task.complete result=Failed;done=true;]
Finishing: Check Dependencies
```

![image](https://user-images.githubusercontent.com/51720070/72636015-01779f00-3913-11ea-841c-4f8c235741a5.png)

--- a/news/3 Code Health/9646.md
+++ b/news/3 Code Health/9646.md
@@ -1,0 +1,1 @@
+CI failure on Data science memoize-one dependency being removed.

--- a/package.datascience-ui.dependencies.json
+++ b/package.datascience-ui.dependencies.json
@@ -124,7 +124,6 @@
 	"markdown-escapes",
 	"material-colors",
 	"mdast-add-list-metadata",
-	"memoize-one",
 	"monaco-editor",
 	"monaco-editor-textmate",
 	"monaco-textmate",

